### PR TITLE
feat: Add styling to event card

### PR DESCRIPTION
**Pull request by CalCorbin**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
This PR updates the styling for the EventCard component to match what is seen in Figma.
image

* **What is the current behavior?** (You can also link to an open issue here)
https://github.com/codefortulsa/website/issues/53

* **What is the new behavior (if this is a feature change)?**
It looks cooler.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No.

* **Other information**:

### CODE REVIEW

File: src/features/events/EventCard.tsx [hunk 1]
```diff
@@ -6,15 +6,17 @@ export type CFTEvent = {
...
 
 const EventCard = ({ event }: { event: CFTEvent }) => {
+	const title: string = event.title.toUpperCase();
+
 	return (
-		<article className='bg-cft-light-gray p-6 text-cft-black'>
-			<h1>{event.title}</h1>
-			<h2>{event.date}</h2>
-			<h2>{event.location}</h2>
+		<article className='bg-[#133A4E] text-white p-4 shadow-lg shadow-red-900'>
```
Comment (zenlex) [inline src/features/events/EventCard.tsx:12]: add the dark:prefix to colors and add the light theme colors as default values. 
also looks like  #133A43 is set in tailwind.config.js as `cft-blue`. Looking at the actual color values, lets rename it to '`cft-bluegreen` in both places and lose the bracket notation.

File: src/features/events/EventCard.tsx [hunk 2]
```diff
@@ -6,15 +6,17 @@ export type CFTEvent = {
...
+			<div className='text-3xl'>{title}</div>
```
Comment (zenlex) [inline src/features/events/EventCard.tsx:13]: lets make this an h2 tag for screen readers. I'll end up adding an h1 'events' title on the page for same reason.

File: src/features/events/EventCard.tsx [hunk 3]
```diff
@@ -6,15 +6,17 @@ export type CFTEvent = {
...
+			<div className='text-2xl'>{event.date}</div>
```
Comment (zenlex) [inline src/features/events/EventCard.tsx:14]: lets use `<time>` tag here

**Review by zenlex — CHANGES_REQUESTED** [review]

File: src/features/events/EventCard.tsx (hunk 2 above)
Comment (CalCorbin) [reply src/features/events/EventCard.tsx]: Done.

**Pull request merged**